### PR TITLE
update email and talks

### PR DESCRIPTION
**Pull request by LaurentTardif**
Bonjour,

je viens de mettre à jour mon email et les talls proposés

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -2990,85 +2990,53 @@ var data = {
...
+            "Kanban"
           ],
           "lang": [
             "fr"
           ]
         },
         {
-          "title": "Apache Flink : la 4eme Génération de Processing est la:",
-          "abstract": "Introduction à Flink.<br/>- Présentation ou discussions.<br/>- Découverte ou approfondissements",
+          "title": "Devops:",
```
Comment (dadoonet) [inline baggers.js]: `Devops:` is really the title of the talk or you are missing something?

File: baggers.js [hunk 2]
```diff
@@ -2990,85 +2990,53 @@ var data = {
...
-            "Agile",
-            "Scrum",
-            "Kanban",
-            "Flink",
-            "Intégration continue",
-            "Test",
-            "Tests unitaires",
-            "Usine logicielle",
+            "Docker",
+			"Jenkins",
```
Comment (dadoonet) [inline baggers.js]: Could you please fix the indentation?

File: baggers.js [hunk 3]
```diff
@@ -2990,85 +2990,53 @@ var data = {
...
             "Industrialisation"
           ],
           "lang": [
             "fr"
           ]
         },
         {
-          "title": "Devops & Integration Continue :",
-          "abstract": "Outils et méthodes pour les équipes de développement qui veulent réussir dans l'agilité<br/>- Présentation & discussions",
+          "title": "Kubernetes:",
```
Comment (dadoonet) [inline baggers.js]: Same as previous. Why this `:` character which seems to tell that you wanted to add something.

File: baggers.js [hunk 4]
```diff
@@ -2990,85 +2990,53 @@ var data = {
...
+            "Kubernetes"
           ],
           "lang": [
             "fr"
           ]
         },
-        {
-          "title": "Tests automatiques et specification executable : la pratique :",
-          "abstract": "Quelles strategies de tests sont possibles, et que sont les specifications executables",
+		{
```
Comment (dadoonet) [inline baggers.js]: Indentation?

File: baggers.js [hunk 5]
```diff
@@ -2990,85 +2990,53 @@ var data = {
...
+          "title": "Docker:",
```
Comment (dadoonet) [inline baggers.js]: Same for `:`

File: baggers.js [hunk 6]
```diff
@@ -2990,85 +2990,53 @@ var data = {
...
-            "Tests unitaires",
-            "Usine logicielle",
-            "Industrialisation"
+            "Docker"
+          ],
+          "lang": [
+            "fr"
+          ]
+        },
+		{
```
Comment (dadoonet) [inline baggers.js]: Indentation?

File: baggers.js [hunk 7]
```diff
@@ -2990,85 +2990,53 @@ var data = {
...
+          "title": "Kafka:",
```
Comment (dadoonet) [inline baggers.js]: Same for `:`

**Review by dadoonet — CHANGES_REQUESTED** [review]
I left some questions and remarks.
Could you fix the indentation problems please?

**Comment by LaurentTardif** [comment]
Salut David, j'ai corrigé l'indentation et changé un peu le titre pour devops pour le rendre un peu plus parlant.
Merci

**Review by dadoonet — APPROVED** [review]

**Pull request merged**